Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -98,6 +98,7 @@ function mac_process_gpu_entitlements()
 
         if (( "${TARGET_MAC_OS_X_VERSION_MAJOR}" >= 110000 ))
```

Comment:
We don't need these checks, older OSes aren't supported any more.

Reply:
> We don't need these checks, older OSes aren't supported any more.

Ah, good point! I will remove.

Thanks for reviewing!